services/screen_fetcher.rb

```diff
@@ -1,15 +1,25 @@
 class ScreenFetcher
   class << self
-    def call
+    attr_reader :base64
+
+    def call(base64: false)
+      @base64 = base64
       last_generated_image || empty_state_image
     end
 
     def last_generated_image
-      img_path = Dir.glob(File.join(base_path, '*.*')).max { |a, b| File.ctime(a) <=> File.ctime(b) }
-      return nil unless img_path
+      full_img_path = Dir.glob(File.join(base_path, '*.*')).max { |a, b| File.ctime(a) <=> File.ctime(b) }
+      return nil unless full_img_path
+
+      filename = full_img_path.split('/').last # => 1as4ff.bmp
```

Comment:
```suggestion
      filename = File.basename(full_img_path) # => 1as4ff.bmp
```
More idiomatic and Windows-friendly